### PR TITLE
Feature process existing user invite emails

### PR DESCRIPTION
**Pull request by justinl-y**
Closes issue #87

### CODE REVIEW

File: js/components/InviteForm/index.js [hunk 1]
```diff
@@ -89,7 +89,11 @@ class InviteForm extends Gandalf {
     this.setState({ emailCount: this.numberOfValidEmails() });
 
     // Submit to REDUX
-    console.log(data);
+    const encodedGuestEmails = encodeObjectValues(data);
+    // console.log(decodeObjectValues(encodeObjectValues(data)));
```
Comment (BlakeTurner) [inline js/components/InviteForm/index.js]: !!!

File: js/components/InviteForm/index.js [hunk 2]
```diff
@@ -89,7 +89,11 @@ class InviteForm extends Gandalf {
...
+
+    console.log(encodedGuestEmails);
```
Comment (BlakeTurner) [inline js/components/InviteForm/index.js]: 👊

**Review by BlakeTurner — CHANGES_REQUESTED** [review]
Code looks good, but you have to clean up the unused stuff

**Comment by BlakeTurner** [comment]
FYI Conflicts have arisen

File: js/components/InviteForm/index.js (hunk 1 above)
Comment (justinl-y) [reply js/components/InviteForm/index.js]: Done.

File: js/components/InviteForm/index.js (hunk 2 above)
Comment (justinl-y) [reply js/components/InviteForm/index.js]: Done.

**Comment by justinl-y** [comment]
Conflicts and issues resolved.

**Review by BlakeTurner — APPROVED** [review]

**Pull request merged**